blockchain/indexers/slpindex_test.go

```diff
@@ -70,7 +70,7 @@ func TestSlpInputUnitTests(t *testing.T) {
 			if err != nil || slpMsg == nil {
 				continue
 			}
-			tokenID, err := goslp.GetSlpTokenID(tx)
+			tokenID, _ := goslp.GetSlpTokenID(tx)
```

Comment:
We should at least have a comment why we are not handling the error here. =)

Reply:
Good point, I've updated to just have the test fail if there's an error.. it was much easier than trying to explain it away.